source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -32,24 +33,39 @@ def __init__(self, obj, position, _rangeObj=None):
 
 
 class winConsoleUIA(Terminal):
+	STABILIZE_DELAY = 0.03
 	_TextInfo = consoleUIATextInfo
 	_isTyping = False
 	_lastCharTime = 0
 	_TYPING_TIMEOUT = 1
 
 	def _reportNewText(self, line):
 		# Additional typed character filtering beyond that in LiveText
-		if self._isTyping and time.time() - self._lastCharTime <= self._TYPING_TIMEOUT:
+		if (
+			self._isTyping
+			and time.time() - self._lastCharTime <= self._TYPING_TIMEOUT
+		):
 			return
 		super(winConsoleUIA, self)._reportNewText(line)
 
 	def event_typedCharacter(self, ch):
 		if not ch.isspace():
 			self._isTyping = True
+		if ch in ('\r', '\t'):
```

Comment:
How about `\n`

Reply:
I haven't seen the console send a \n on return, but there's no reason why I couldn't add it...